docs/user_guide/Client_Management.rst

```diff
@@ -1,10 +1,14 @@
 .. _Client_Management:
 
+
 Client Management
 =================
 
-This chapter provides guidance to install and configure SIMP clients
-based on the standard SIMP system installed using the SIMP DVD.
```

Comment:
This chapter provides guidance to install and configure SIMP clients, via kickstart, with the resources supplied by the SIMP ISO.